asv_bench/benchmarks/utils/common.py

```diff
@@ -504,20 +504,14 @@ def execute(
             return
 
         # compatibility with old Modin versions
-        all(
-            map(
-                lambda partition: partition.drain_call_queue() or True,
-                partitions,
-            )
-        )
         if ASV_USE_ENGINE == "ray":
             from ray import wait
 
-            all(map(lambda partition: wait([partition._data]), partitions))
+            all(map(lambda partition: wait([partition.list_of_blocks[0]]), partitions))
         elif ASV_USE_ENGINE == "dask":
             from dask.distributed import wait
 
-            all(map(lambda partition: wait(partition._data), partitions))
+            all(map(lambda partition: wait(partition.list_of_blocks[0]), partitions))
```

Comment:
ditt about waiting for all blocks.

Reply:
Use single way for ray/dask/python `ASV_USE_ENGINE` using the partition manager's `wait_partitions`